test/fixed_rank.jl

```diff
@@ -9,36 +9,70 @@ include("utils.jl")
     v = UMVTVector([0.0 0.0; 0.0 0.0; 1.0 1.0], [1.0 0.0; 0.0 1.0], zeros(2, 2))
     @test repr(M) == "FixedRankMatrices(3, 2, 2, ℝ)"
     @test repr(Mc) == "FixedRankMatrices(3, 2, 2, ℂ)"
-    @test sprint(show, "text/plain", x) == """
-    SVDMPoint{Array{Float64,2},Array{Float64,1},Array{Float64,2}}
-    U factor:
-     3×2 Array{Float64,2}:
-      1.0  0.0
-      0.0  1.0
-      0.0  0.0
-    singular values:
-     2-element Array{Float64,1}:
-      1.0
-      1.0
-    Vt factor:
-     2×2 Array{Float64,2}:
-      1.0  0.0
-      0.0  1.0"""
-    @test sprint(show, "text/plain", v) == """
-    UMVTVector{Array{Float64,2},Array{Float64,2},Array{Float64,2}}
-    U factor:
-     3×2 Array{Float64,2}:
-      0.0  0.0
-      0.0  0.0
-      1.0  1.0
-    M factor:
-     2×2 Array{Float64,2}:
-      1.0  0.0
-      0.0  1.0
-    Vt factor:
-     2×2 Array{Float64,2}:
-      0.0  0.0
-      0.0  0.0"""
+    if VERSION >= v"1.6.0-DEV.430"
```

Comment:
This got quite long, could we maybe generate this string, such tthat we check that from "U factor:" onwards the lines are queal to the SVD print? Maybe that would be a better test.

Reply:
I'll try that.

Reply:
I tried that and there are subtle differences, I'm not sure it's worth figuring out how to generate that from SVD print.

Reply:
Ok, if this works, we can leave it for now, if I find time I can still try my advanced idea (which would get us independent of changes wrt to SVD printings).